src/cheri/riscv-hybrid-integration.adoc

```diff
@@ -135,79 +117,19 @@ possibly a <<root-rw-cap>> capability).
 .Unprivileged default data capability register
 include::img/ddcreg.edn[]
 
-// This depends on the priv spec, can't mention it here
-// NOTE: <<section_cheri_disable,CHERI register and instruction access>> is not required for the implicit access required by checking memory accesses against <<ddc>>.
-//
-// NOTE: <<ddc>> is a data pointer, so it does not need to be able to hold all possible invalid addresses (see <<section_invalid_addr_conv>>).
-
-=== {cheri_default_ext_name} Memory Accesses
-
-In {cheri_cap_mode_name} all memory accesses behave as defined by <<rv32y,{cheri_base_ext_name}>> and are authorised by the base register (`{cs1}`), from which the bounds and permissions are taken. For example:
-
-* `lw t0, 16({abi_creg}a0)`
-
-In {cheri_int_mode_name} the `lw` has the same behavior and assembly syntax as the RV32I/RV64I:
-
-* `lw t0, 16(a0)`
-
-All memory accesses must still be checked, but now they are authorized by the capability in <<ddc>> instead.
-
-=== {cheri_default_ext_name} Branches and Jumps
-
-The rules from xref:section_existing_riscv_insns[xrefstyle=short] are applied, so that all branch and jump targets are checked against <<pcc>>.
-
-NOTE: In {cheri_int_mode_name} JALR does _not_ copy the target capability register `{cs1}` into <<pcc>>, instead it updates the address of <<pcc>> with the target value.
-The only difference to RV32I/RV64I without {cheri_default_ext_name}, is that the target address is checked against the bounds of <<pcc>>.
-
-[#section_hybrid_ext_instructions]
-=== {cheri_default_ext_name} Instructions
-
-{cheri_default_ext_name} introduces new instructions to switch CHERI execution modes.
-Additionally, {cheri_default_ext_name} makes all instructions defined by {cheri_default_ext_name} available.
-
-.Instructions summary for {cheri_default_ext_name}
-[#tab_hybrid_summary,%autowidth,options=header,align="center",cols="1,4"]
-|=======================
-|Mnemonic     |Description
-|<<SCMODE>>    |Set capability CHERI execution mode (<<m_bit>>)
-// The output is written to an unprivileged general purpose register, not <<pcc>>.
-|<<GCMODE>>    |Get capability CHERI execution mode (<<m_bit>>)
```

Comment:
We completely dropped any mention of SCMODE/GCMODE here. We should restore that

Reply:
ok - restored.